cli/flox-rust-sdk/src/models/container_builder.rs

```diff
@@ -0,0 +1,40 @@
+use std::io::{self, Write};
+use std::path::PathBuf;
+use std::process::{Command, Stdio};
+
+use thiserror::Error;
+
+pub struct ContainerBuilder {
+    path: PathBuf,
+}
```

Comment:
style/nonblocking: do we really gain anything by making this a struct instead of just having a `stream_container` helper function in `commands/environment.rs`? I'm not too worried about passing something other than a container builder script to the helper function, but even then we could just have `type ContainerBuilder = PathBuf`

Reply:
I prefer wrapping this.
We do not _need_ it for safety reasons.
Then again, its zero-cot at runtime, its a concrete "thing" representing the script on disk that we can attach `stream_container` to (rather than having a free function), and yeah makes it harder to 'confuse' the the inputs..

Reply:
If I see a free function that takes a `PathBuf` and a sink, I immediately know exactly what it's doing. If I see a `ContainerBuilder::stream_container` method, it's less obvious to me personally what it's doing. I'm not concerned about cost at runtime; I'm concerned about readability.